bin/train_llm.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
```

Comment:
it feels that this has a lot of similarities with the other scripts. It might be a better idea to split those into scripts that download the data and the scripts that actually can be used to train the LLVM